spec/main/reportToRow.spec.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const fs = require('fs');
+const {reportToRow, reportToRowHeaders} = require('../../reportToRow');
+const testDesktopCsvPath = path.join(
+  __dirname, 
+  "../../", 
+  'spec', 
+  'helpers', 
+  'lighthouse', 
+  '7_15_2020_6_15_05PM', 
+  'tgiles.github.io_web-resume.html.desktop.report.csv'
```

Comment:
To reduce brittleness, this test should probably use it's own mocked report. (I reused and existing one to save time.)